org.eclipse.dawnsci.analysis.dataset/src/org/eclipse/dawnsci/analysis/dataset/mask/MaskCircularBuffer.java

```diff
@@ -23,70 +24,34 @@
 public class MaskCircularBuffer {
 
 	private IntegerDataset mask;
-	private int[] shape;
 	int bitMask = START;
 	private static final int END = 0x80000000;
 	private static final int START = 0x00000001;
 	
+	private final static Logger logger = LoggerFactory.getLogger(MaskCircularBuffer.class);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)